Lib/fontbakery/profiles/googlefonts_conditions.py

```diff
@@ -265,26 +265,63 @@ def familyname(font):
 
 
 @condition
-def ttfautohint_stats(font):
+def hinting_stats(font):
+  """
+  Return file size differences for a hinted font compared to an dehinted version of same file
+  """
   from ttfautohint import ttfautohint, libttfautohint
   from io import BytesIO
   from fontTools.ttLib import TTFont
+  from fontTools.subset import main as pyftsubset
   from fontbakery.profiles.shared_conditions import is_ttf
-
-  if not is_ttf(TTFont(font)):
+  from fontbakery.profiles.shared_conditions import is_cff
+  from fontbakery.profiles.shared_conditions import is_cff2
+
+  if is_ttf(TTFont(font)):
+    original_buffer = BytesIO()
+    TTFont(font).save(original_buffer)
+    dehinted_buffer = ttfautohint(in_buffer=original_buffer.getvalue(),
+                                  dehint=True)
+    dehinted_size = len(dehinted_buffer)
+    version = libttfautohint.version_string
+
+  elif is_cff(TTFont(font)) or is_cff2(TTFont(font)):
+    original_buffer = BytesIO()
+    TTFont(font).save(original_buffer)
```

Comment:
`original_buffer` does not seem to be actually needed in the CFF case

Reply:
Ah yes, correct. Oversight when combining my code with the existing check.